docs/dg/dev/backend-development/data-manipulation/data-publishing/implement-publish-and-synchronization.md

```diff
@@ -308,7 +299,7 @@ For performance considerations, events are passed to the listener in bulk. Even
 
 {% endinfo_block %}
 
-Also, you need to configure listener classes in `PublisherDependencyProvider::getPublisherPlugins()` to enable them. The listeners are listening to the default publish queue. Alternatively, the mapping of the listener class can adjust the listened queue in its key. For details, see the full code in `data/shop/development/current/vendor/spryker/publisher/src/Spryker/Zed/Publisher/PublisherDependencyProvider::getPublisherPlugins()`:
+Also, you need to wire listener classes in `PublisherDependencyProvider::getPublisherPlugins()` to enable them. The listeners are listening to the default publish queue. Alternatively, the mapping of the listener class can adjust the listened queue in its key.:
```

Comment:
is this something that was mentioned above, or we mention this configuration just there?